### ISSUE TITLE
App crashes on connection

### ISSUE THREAD

**Issue by nathanhornby** [jeromelebel/MongoHub-Mac#212]
Haven't really got far with the app I'm afraid.

I've dropped in my connection details - but as soon as the app connects it just crashes.

Looking at the `mongod` log this is all I'm getting:

```
2015-03-20T09:57:09.906-0400 I NETWORK  [initandlisten] connection accepted from XXXXXX #2 (1 connection now open)
2015-03-20T09:57:09.973-0400 I NETWORK  [conn2] end connection XXXXXX (0 connections now open)
```

**Comment by jeromelebel** [comment]
can you give me the crash log too?

**Comment by nathanhornby** [comment]
Sorry - this happened in the office and I'm out for the weekend now. I'll try and remember to sort it first thing Monday morning!

**Comment by jeromelebel** [comment]
no problem. As long as you can reproduce it all the time that should be easy to fix.

**Comment by sujal** [comment]
MongoHub crashes every time connecting to a 3.0.x Mongo instance. Here is a crash log: http://cl.ly/3w1b2O2I3P1E

**Comment by nathanhornby** [comment]
Thanks Sujal, completely forgot about this issue!

**Comment by jeromelebel** [comment]
According to this crash log, this issue is the same than https://github.com/jeromelebel/MongoHub-Mac/issues/200

It has been fixed in the beta version, and will be fixed in the next version.

**Comment by jeromelebel** [comment]
Should be fixed in 3.1. I let you verify this issue and close it.

**Comment by sujal** [comment]
Yup - I just tried 3.1.3 and am able to connect and access a 3.x MongoDB instance.

Thanks!
